tripal/tests/src/Functional/Services/TripalFieldCollectionTest.php

```diff
@@ -15,6 +15,14 @@
  */
 class TripalFieldCollectionTest extends TripalTestBrowserBase {
 
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'tripal_test',
```

Comment:
Doesn't it also need to include tripal?
```suggestion
    'tripal',
    'tripal_test',
```